eth/handler.go

```diff
@@ -473,6 +461,28 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 		}
 		return p.SendBlockBodiesRLP(bodies)
 
+	case p.version >= eth62 && msg.Code == BlockBodiesMsg:
+		// A batch of block bodies arrived to one of our previous requests
```

Comment:
Is it possible this code is executed if a BlockBodiesMsg is received which is not the result of previous requests?